promql/engine.go

```diff
@@ -1561,6 +1561,13 @@ func (ev *evaluator) eval(expr parser.Expr) (parser.Value, storage.Warnings) {
 					return ev.VectorBinop(e.Op, v[0].(Vector), v[1].(Vector), e.VectorMatching, e.ReturnBool, sh[0], sh[1], enh), nil
 				}, e.LHS, e.RHS)
 			}
+		case lt == parser.ValueTypeScalar && rt == parser.ValueTypeMatrix:
+			outmat, warnings := ev.RangeVectorscalarbinop(e.Op, e.LHS, e.RHS, true)
+			return outmat, warnings
```

Comment:
Just a suggestion, you can merge 2 lines of code into single line since the return values on line 1566 are exactly same that func at 1565 is yielding. Simply use: 
```return ev.RangeVectorscalarbinop(e.Op, e.LHS, e.RHS, true)```